tests/suites/test_suite_ssl.function

```diff
@@ -137,6 +137,99 @@ int mbedtls_test_buffer_get( mbedtls_test_buffer *buf,
     return output_len;
 }
 
+/*
+ * Errors used in the message transport mock tests
+ */
+ #define MBEDTLS_TEST_ERROR_QUEUE_NULL -11
```

Comment:
For commit "Add a message metadata queue in ssl tests": MBEDTLS_TEST_ERROR_QUEUE_NULL is renamed to ARG_NULL in the next commit. Consider adding it correctly the first time. This will make the diff for the next commit less noisy and easier to read.